appcues/src/main/java/com/appcues/debugger/DebuggerViewModel.kt

```diff
@@ -225,7 +226,7 @@ internal class DebuggerViewModel(
 
     fun captureScreen(debuggerState: MutableDebuggerState) {
         appcuesCoroutineScope.launch {
-            experienceRenderer.dismissCurrentExperience(markComplete = false, destroyed = false)
+            experienceRenderer.stop()
```

Comment:
not sure if this impacts anything, instead of calling dismissCurrentExperience I figured we could call stop instead before capturing the screen.

internally I noticed the difference is that stop() calls  action with "destroyed = yes". when destroyed is true it means there was an external exit from the experience and we should not rely on UI to move the StateMachine, I think its safe to change to destroyed = false since we are in control here.

Reply:
the other alternative I think would be to close only the Modal experience, not sure whats correct here.